src/Tools/ExternalAccess/FSharp/Microsoft.CodeAnalysis.ExternalAccess.FSharp.csproj

```diff
@@ -17,14 +17,16 @@
   
   <ItemGroup>
     <!--
-      ⚠ ONLY F# ASSEMBLIES MAY BE ADDED HERE ⚠
+      ⚠ ONLY F# ASSEMBLIES MAY BE ADDED HERE ⚠ - Will be removed soon once all IVTs are removed.
```

Comment:
> - Will be removed soon once all IVTs are removed. [](start = 46, length = 50)

I think this one should actually stay - this should be the only IVT to F# we have in Roslyn code base.

Reply:
@tmat is correct here. These IVTs will not be removed unless they are not needed by F#.